src/components/shared/Validations.tsx

```diff
@@ -1,90 +1,169 @@
-import { safeStringify } from '@stoplight/json';
-import { Dictionary } from '@stoplight/types';
-import { Popover } from '@stoplight/ui-kit';
+import { RegularNode } from '@stoplight/json-schema-tree';
+import { Flex, Text } from '@stoplight/mosaic';
+import { Dictionary, Primitive } from '@stoplight/types';
+import { capitalize, keys, omit, pick, pickBy, uniq } from 'lodash';
```

Comment:
`keys` - any reason we can't just use `Object.keys`?

Reply:
Only to be consistent with other lodash functions that we needed to use.